src/shared/store/actions/utils/errors.ts

```diff
@@ -0,0 +1,33 @@
+export class Forbidden extends Error {
+  readonly code: number = 403;
```

Comment:
Same remark about numbers, they should be constants with names, like HTTP_STATUS_UNAUTHORIZED